ProjectStructureAndRoles.md

```diff
@@ -75,17 +70,17 @@ A Project participant may resign by notifying the team mailing list and by sendi
 
 **Working Group participants**
 
-Each Project participant can be part of one or more of the Working Groups of the Project by adding his name to the WG_PARTICIPANTS.MD files. By that he gets the role Working Group participant.
+Each Project participant can be part of one or more of the Working Groups of the Project by subscribing to the mailing list(s). By that he gets the role Working Group participant, the WG_PARTICIPANTS.MD file(s) is/are then also adjusted by the administrators of the Project.
 
-A Working Group participant may resign by removing his name of the WG_PARTICIPANTS.MD file.
+A Working Group participant may resign by unsubscribing from the mailing list(s), the WG_PARTICIPANTS.MD file(s) is/are adjusted by the administrators of the Project.
 
 **Contributors**
 
-Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. There’s no formal process for becoming or resigning as Contributor. The list of Contributors of a specific Sub Project is maintained by GitHub.
+Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. To become or resign as Contributor please subscribe / unsubscribe to the mailing list of the Sub Project. The list of Contributors of a specific Sub Project is maintained by the administrators of the Project using GitHub functionality.
 
 **Codeowners**
 
-Codeowners can merge code and accept pull requests. A Codeowner is responsible for the overall quality and stewardship of the Project. Contributors can get a Codeowner role if they are deeply involved in contributing code, pull request review, and triaging issues in the Sub Project for a minimum of three months and perform an ongoing contribution. Codeowners are decided by the Maintainers and formalized by changing the CODEOWNERS file. Changes in codeownership have to be announced on the CAMARA Project mailing list.
+Codeowners can merge code and accept pull requests. A Codeowner is responsible for the overall quality and stewardship of the Project. Contributors can get a Codeowner role if they are Contributors and deeply involved in contributing code, pull request review, and triaging issues in the Sub Project for a minimum of three months and perform an ongoing contribution. Codeowners are decided by the Maintainers and formalized by changing the CODEOWNERS file. Changes in codeownership have to be announced on the CAMARA Project mailing list.
```

Comment:
why a minimum of three months?
What about saying " A Codeowner is responsible for the overall quality and stewardship of the Project. Contributors can get a Codeowner role if they are Contributors and deeply involved in contributing code, pull request review, perform an ongoing contribution. Codeowners are decided by the [...]"